src/routine.cpp

```diff
@@ -24,6 +24,7 @@ namespace clblast {
 // The constructor does all heavy work, errors are returned as exceptions
 Routine::Routine(Queue &queue, EventPointer event, const std::string &name,
                  const std::vector<std::string> &routines, const Precision precision,
+                 std::initializer_list<const char *> source_pct,
```

Comment:
Ah, PadCopyTranspose :-) Perhaps this is no longer needed then if you don't want to treat that separately?